composed_api.rst

```diff
@@ -545,33 +839,69 @@ Request Parameters
-+-------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| cell\_spacing           | *Optional*. Used together with ``generate\_dem``. Default: 1.0                                                                           |
-+-------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| pcl\_terrain            | *Optional*. Used to trigger a PMF Bare Earth export. Accepts ``ubran``, ``suburban``, ``mountainous``, and ``foliated``. Default: None   |
-+-------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| sri\_hres               | *Optional*. Used to trigger a Sarnoff Bare Earth export. Accespts the horizontal resolution. Default: None                               |
-+-------------------------+------------------------------------------------------------------------------------------------------------------------------------------+
++------------------+--------------------------------------------------------+
+| Query parameter  | Value                                                  |
++==================+========================================================+
+| source           | *Required*. Your GRiD generated API key.               |
++------------------+--------------------------------------------------------+
+| products         | *Required*. A list of product primary keys to include  |
+|                  | in the export, separated by ``+`` or ``,``.            |
++------------------+--------------------------------------------------------+
+| name             | *Optional*. An optional name for the export.           |
++------------------+--------------------------------------------------------+
+| intensity        | *Optional*. Whether or not to export intensity.        |
+|                  | Default: True.                                         |
++------------------+--------------------------------------------------------+
+| dim\_classificat | *Optional*. Whether or not to export classification.   |
+| ion              | Default: True.                                         |
++------------------+--------------------------------------------------------+
+| hsrs             | *Optional*. Accepts an EPSG code. Defaults to AOI SRS. |
++------------------+--------------------------------------------------------+
+| file\_export\_op | *Optional*. Determine file merging strategy. Accepts   |
+| tions            | ``individual`` and ``collect``. Default:               |
+|                  | ``individual``.                                        |
++------------------+--------------------------------------------------------+
+| export\_file\_ty | *Optional*. Determine the The format of the output     |
```

Comment:
For raster export, the equivalent key is `file_format_options`. I like that better than `export_file_type` because 1) it is clear that it is specifying the format, and 2) it is closer to it's sister key `file_export_options`. (I don't know why it bothers me to see one called `file_export_` and the other `export_file_` - it just felt convoluted).